client/pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: spine_client
 description: Dart-based library for client applications of Spine-based systems.
-version: 0.1.9
+version: 0.2.0
```

Comment:
I wonder how close are we to supporting the features of, say, 1.6.x, so that we could set the same 1.6.x version here?

Reply:
I suggest we change the version to match the global Spine version when the Java-like high-level client API is implemented.